lib/config.ts

```diff
@@ -55,12 +55,34 @@ const ethereum = {
   emailSubjectPrefix: '[Ethereum]:',
 };
 
+const optimism = {
+  ...baseConfig,
+  centerNetwork: '',
+  nftBackedLoansSubgraph:
+    'https://api.thegraph.com/subgraphs/name/with-backed/backed-protocol-optimism',
+  nftSalesSubgraph:
+    'https://api.thegraph.com/subgraphs/name/adamgobes/nft-sales-indexer',
+  eip721Subgraph:
+    'https://api.thegraph.com/subgraphs/name/optimism-erc721-subgraph',
```

Comment:
@adamgobes let's double check this, doesn't seem to be valid

Reply:
ah oops had the change staged but not commited 😭